### PR TITLE
Added input box and tidyup for javascript webbook

### PR DESCRIPTION
**Pull request by naimishvora-macrometa**
- Added input box for javascript webbook
- added tidyup section in some webbooks

### CODE REVIEW

**Review by alan-macrometa — COMMENTED** [review]
did you manage to make the password boxes obscured? e.g. password = **********

File: javascript/08_stream_worker(javascript).ipynb [hunk 1]
```diff
@@ -118,11 +166,11 @@
     "const jsc8 = require(\"jsc8\");\n",
     "\n",
     "// ----- simple way  -----\n",
-    "const client = new jsc8(global_url);\n",
+    "const client = new jsc8(fed_url);\n",
```
Comment (jyuwei) [inline javascript/08_stream_worker(javascript).ipynb:169]: Please also fix the "global_url" in the commented section below, on line 179 and 186

**Review by alan-macrometa — APPROVED** [review]

**Comment by alan-macrometa** [comment]
assuming you are doing the same for Python, and CLI?

**Comment by naimishvora-macrometa** [comment]
> assuming you are doing the same for Python, and CLI?

Next I am doing the same for python

**Pull request merged**